lib/gollum-lib/markups.rb

```diff
@@ -1,20 +1,58 @@
 # ~*~ encoding: utf-8 ~*~
 module Gollum
+  module MarkupRegisterUtils
+    # Check if a gem exists. This implementation requires Gem::Specificaton to
+    # be filled.
+    def gem_exists? name
+      return Gem::Specification.find {|x| x.name == name} != nil
```

Comment:
Explicit return statement is not needed.
Also, please choose a more descriptive variable name than `x` like `gem` or `gemspec`.